tests/test_api_endpoints.py

```diff
@@ -1158,3 +1158,57 @@ def test_visit_maps_read_visits_exception(
     assert "Database connection error" in response.json()["detail"]
 
     app.dependency_overrides.pop(get_access_token, None)
+
+
+def test_expected_exposures_endpoint(monkeypatch):
+    endpoint = "/expected-exposures?dayObsStart=20240101&dayObsEnd=20240102"
+
+    dummy_expected_exposures = {"sum": 220}
+
+    # Patch get_expected_exposures to return dummy data
+    monkeypatch.setattr(
+        "lsst.ts.logging_and_reporting.web_app.main.get_expected_exposures",
+        lambda dayObsStart, dayObsEnd: dummy_expected_exposures,
+    )
+
+    # Success path
+    response = client.get(endpoint)
+    assert response.status_code == 200
+
+    data = response.json()
+    assert "sum_exposures" in data
+    assert data["sum_exposures"] == dummy_expected_exposures["sum"]
+
+    # Error path (generic exception)
+    def raise_error(dayObsStart, dayObsEnd):
+        raise Exception("failure")
+
+    monkeypatch.setattr(
+        "lsst.ts.logging_and_reporting.web_app.main.get_expected_exposures",
+        raise_error,
+    )
+
+    response = client.get(endpoint)
+    assert response.status_code == 500
+    assert response.json()["detail"] == "failure"
+
+
+def test_expected_exposures_endpoint_passes_correct_params(monkeypatch):
+    endpoint = "/expected-exposures?dayObsStart=20240101&dayObsEnd=20240102"
+
+    called = {}
+
+    def fake_service(dayObsStart, dayObsEnd):
+        called["start"] = dayObsStart
+        called["end"] = dayObsEnd
+        return {"sum": 10}
```

Comment:
```suggestion
    def fake_service(day_obs_start, day_obs_end):
        called["start"] = day_obs_start
        called["end"] = day_obs_end
        return {"sum": 10}
```
We need to standardize our `snake_case` use, but please let's start with tests (here and everywhere else) 🙏 .